src/transaction/replication.c

```diff
@@ -449,20 +448,32 @@ repl_log_insert (THREAD_ENTRY * thread_p, const OID * class_oid, const OID * ins
       LOG_REPL_RECORD *recsp = tdes->repl_records;
       int i;
 
-      for (i = 0; i < tdes->fl_mark_repl_recidx; i++)
+      if (strcmp (class_name, CT_SERIAL_NAME) != 0)
```

Comment:
I think it's good but is it really ok? 
I mean.. is only 'db_serial' catalog class related with this problem?
if it is, this line is OK

Reply:
So, the new code was used in case of db_serial only. I hope it will be revealed in the regression test if there exist any problem.